dataflow/src/main/java/org/checkerframework/dataflow/busyexpression/BusyExprValue.java

```diff
@@ -0,0 +1,46 @@
+package org.checkerframework.dataflow.busyexpression;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.dataflow.cfg.node.BinaryOperationNode;
+
+/**
+ * BusyExprValue class contains a BinaryOperationNode. So we only consider expressions that are in
```

Comment:
What is the point of the class? Equality comparison is implemented differently?

Reply:
Added it